scripts/docker-service.sh

```diff
@@ -2,4 +2,4 @@
 
 # zenoss-inspector-tags docker
 
-cat /usr/lib/systemd/system/docker.service
+cat /etc/systemd/system/docker.service.d/docker.conf || cat /usr/lib/systemd/system/docker.service
```

Comment:
Zenoss documentation instructs the user to customize `/usr/lib/systemd/system/docker.service`. 
If they want to drop a file in `/etc/systemd/system/docker.service.d` that's supported by systemd and recommended by docker, but who decides or recommends that the file should be named `docker.conf`?